services/Skyra.Database/Migrations/20210713150528_V06_EventsModernization.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Skyra.Database.Migrations
+{
+    public partial class V06_EventsModernization : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
+                name: "events.member-add",
+                table: "guilds");
+
+            migrationBuilder.DropColumn(
+                name: "events.member-nickname-update",
+                table: "guilds");
+
+            migrationBuilder.DropColumn(
+                name: "events.member-remove",
+                table: "guilds");
+
+            migrationBuilder.DropColumn(
+                name: "events.member-role-update",
+                table: "guilds");
+
+            migrationBuilder.DropColumn(
+                name: "events.message-delete",
+                table: "guilds");
+
+            migrationBuilder.DropColumn(
+                name: "events.message-edit",
+                table: "guilds");
+
+            migrationBuilder.RenameColumn(
+                name: "channels.logs.nsfw-message",
+                table: "guilds",
+                newName: "channels.logs.message-update-nsfw");
+
+            migrationBuilder.RenameColumn(
+                name: "channels.logs.message",
+                table: "guilds",
+                newName: "channels.logs.message-update");
+
+            migrationBuilder.RenameColumn(
+                name: "channels.logs.member",
+                table: "guilds",
+                newName: "channels.logs.message-delete-nsfw");
+
+            migrationBuilder.AddColumn<string>(
+                name: "channels.logs.member-add",
+                table: "guilds",
+                type: "character varying(19)",
+                maxLength: 19,
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "channels.logs.member-nickname-update",
+                table: "guilds",
+                type: "character varying(19)",
+                maxLength: 19,
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "channels.logs.member-remove",
+                table: "guilds",
+                type: "character varying(19)",
+                maxLength: 19,
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
+                name: "channels.logs.member-role-update",
```

Comment:
```suggestion
                name: "channels.logs.member-roles-update",
```

This is triggering when 1 member's 0 or more roles update, therefore roles.